src/Compiler/TaskManager.cpp

```diff
@@ -39,23 +39,18 @@ TaskManager::TaskManager(QObject *parent) : QObject{parent} {
   m_tasks.insert(SYNTHESIS_CLEAN, new Task{"Clean", TaskType::Clean});
   m_tasks.insert(SYNTHESIS_SETTINGS,
                  new Task{"Edit settings...", TaskType::Settings});
-  m_tasks.insert(SYNTHESIS_WRITE_NETLIST, new Task{"Write netlist"});
```

Comment:
Please remove *_WRITE_NETLIST/*_TIMING_REPORT IDs since we don't need them. Also verify loading files from previous version.

Reply:
I discussed this with Ravi and we weren't sure about removing them since then someone could re-claim that ID which would technically conflict with all project saves. I thought of maybe commenting them out and saying that ID# is deprecated. Any opinion on this? Leaving the ID's won't hurt anything, just won't be used right?

Reply:
Note this comment is why i'm trying to keep the ID# claimed/deprecated: https://github.com/os-fpga/FOEDAG/blob/da0771b6c16f717e0462d16e8fd32e515d0c6835/src/Compiler/CompilerDefines.h/#L58-L59

Reply:
okay, let's put a short comment that it is deprecated

Reply:
Sounds good